### PR TITLE
Node release 8.10

### PR DESCRIPTION
**Pull request by dnadales**
# Description

TODO

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated.  These may include:
  - golden tests
  - property tests
  - roundtrip tests
  - integration tests
  See [Runnings tests](https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests) for more details
- [ ] Any changes are noted in the `CHANGELOG.md` for affected package
- [ ] The version bounds in `.cabal` files are updated
- [ ] CI passes. See note on CI.  The following CI checks are required:
  - [ ] Code is linted with `hlint`.  See `.github/workflows/check-hlint.yml` to get the `hlint` version
  - [ ] Code is formatted with `stylish-haskell`.  See `.github/workflows/stylish-haskell.yml` to get the `stylish-haskell` version
  - [ ] Code builds on Linux, MacOS and Windows for `ghc-8.10.7` and `ghc-9.2.7`
- [ ] Self-reviewed the diff

# Note on CI
If your PR is from a fork, the necessary CI jobs won't trigger automatically for security reasons.
You will need to get someone with write privileges.  Please contact IOG node developers to do this
for you.

### CODE REVIEW

File: cardano-node/src/Cardano/Node/Tracing/Era/Shelley.hs [hunk 1]
```diff
@@ -77,6 +77,7 @@ import           Data.Set (Set)
 import qualified Data.Set as Set
 import           Data.Text (Text)
 import qualified Data.Text.Encoding as Text
+import           GHC.IsList (toList)
```
Comment (carbolymer) [inline cardano-node/src/Cardano/Node/Tracing/Era/Shelley.hs:80]:
```suggestion
import           GHC.Exts (toList)
```
Needed for 8.10 support

File: configuration/cardano/mainnet-conway-genesis.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
   "govActionDeposit": 1000000000,
   "dRepDeposit": 2000000,
   "dRepActivity": 20,
+ "minFeeRefScriptCostPerByte": 0,
```
Comment (carbolymer) [inline configuration/cardano/mainnet-conway-genesis.json:27]: I'm unsure about this `0` here. This needs to be updated after benchmarking.

File: cardano-testnet/src/Testnet/Defaults.hs [hunk 3]
```diff
@@ -167,6 +167,7 @@ defaultAlonzoGenesis = do
                            , 1, 1, 69522, 11687, 0, 1, 60091, 32, 196500, 453240, 220, 0, 1, 1, 196500, 453240, 220, 0, 1, 1, 1159724, 392670, 0, 2, 806990
                            , 30482, 4, 1927926, 82523, 4, 265318, 0, 4, 0, 85931, 32, 205665, 812, 1, 1, 41182
                            , 32, 212342, 32, 31220, 32, 32696, 32, 43357, 32, 32247, 32, 38314, 32, 35190005, 10, 57996947, 18975, 10, 39121781, 32260, 10
+                           , 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 -- FIXME GARBAGE VALUE
```
Comment (carbolymer) [inline cardano-testnet/src/Testnet/Defaults.hs:170]: - [ ] update after confirmation from plutus team

File: cardano-testnet/src/Testnet/Defaults.hs [hunk 4]
```diff
@@ -181,7 +182,7 @@ defaultConwayGenesis =
                     , ucppGovActionDeposit = Coin 1_000_000
                     , ucppDRepDeposit = Coin 1_000_000
                     , ucppDRepActivity = EpochInterval 100
-                    , ucppMinFeeRefScriptCostPerByte = undefined
+                    , ucppMinFeeRefScriptCostPerByte = 0 %! 1 -- FIXME GARBAGE VALUE
```
Comment (carbolymer) [inline cardano-testnet/src/Testnet/Defaults.hs:185]: I'm unsure about this `0` here. This needs to be updated after benchmarking. Same applies to genesis files below.

File: cardano-node/src/Cardano/Node/Protocol/Cardano.hs [hunk 5]
```diff
@@ -305,6 +306,7 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
                              (maybe 9 fromIntegral npcTestConwayHardForkAtVersion)
                 Just epochNo -> Consensus.TriggerHardForkAtEpoch epochNo
         }
+      , checkpoints = emptyCheckpointsMap
```
[the remark this answers is not in the document]
Comment (dnadales) [reply cardano-node/src/Cardano/Node/Protocol/Cardano.hs:309]: Absolutely, I missed this one. Ty!

File: cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Babbage/Transaction.hs [hunk 6]
```diff
@@ -93,7 +93,7 @@ hprop_transaction = H.integrationRetryWorkspace 0 "babbage-transaction" $ \tempA
   -- This is the current calculated fee.
   -- It's a sanity check to see if anything has
   -- changed regarding fee calculation.
-  228 H.=== txFee
+  330 H.=== txFee
```
Comment (Jimbo4350) [inline cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Babbage/Transaction.hs:96]: Interesting, the fee increased. Can you put a comment saying that the previous fee was 228 lovelace?

File: cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/KesPeriodInfo.hs [hunk 7]
```diff
@@ -291,7 +292,7 @@ hprop_kes_period_info = H.integrationRetryWorkspace 2 "kes-period-info" $ \tempA
         H.failMessage callStack "cardano-cli query tip returned Nothing for EpochNo"
       Just currEpoch -> return currEpoch
 
-  let nodeHasMintedEpoch = currEpoch + 3
+  let nodeHasMintedEpoch = currEpoch & succ & succ & succ
```
Comment (Jimbo4350) [inline cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/KesPeriodInfo.hs:295]: does `+ 3` not type check?

**Review by Jimbo4350 — COMMENTED** [review]
LGTM. Couple of minor comments.

File: cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Babbage/Transaction.hs (hunk 6 above)
Comment (disassembler) [reply cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Babbage/Transaction.hs]: reverted

File: cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/KesPeriodInfo.hs (hunk 7 above)
Comment (disassembler) [reply cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/KesPeriodInfo.hs]: reverted

**Review by Jimbo4350 — APPROVED** [review]
LGTM! Squash the commits please!

**Pull request merged**